### PR TITLE
Updates to Wallet Page

### PR DESCRIPTION
**Pull request by carsenk**
This adds in a Total Balance at the top of the Wallet page along with the total count of creator coins held and clout balance information. Also added missing "CLOUT" tags, if we want to use a different ticker feel free to lmk @maebeam 

Preview:
walletupdatesfkdljfskldf

### CODE REVIEW

**Comment by carsenk** [comment]
@maebeam @lazynina thoughts on this PR also? Should be good to go if wanted

**Review by lazynina — CHANGES_REQUESTED** [review]
I like these additions to the wallet, but would like to use $CLOUT instead of just CLOUT.  We're trying to move to getting two reviews before merging, so will wait on @maebeam for a second review.

**Review by lazynina — APPROVED** [review]

**Review by maebeam — APPROVED** [review]

**Pull request merged**
